### PR TITLE
Bump npm version to v0.4.0

### PR DESCRIPTION
**Pull request by BSathvik**
## Description

- Published version `0.4.0`
- Updated` js/README.md`

### CODE REVIEW

**Comment by wpank** [comment]
@BSathvik Can you fix these codcov things before merging:

https://app.codacy.com/app/TrustWallet/wallet-core/pullRequest?prid=3921190

File: js/README.md [hunk 1]
```diff
@@ -26,4 +26,24 @@ from { HDWallet } from 'trust-wallet-core';
...
 //=> "army cook device assault vendor case coast rose dismiss lady dust point!"
-```
\ No newline at end of file
+```
+
+## Documentation
+
+### Platforms
+
+`trust-wallet-core` only supports `macOS/OSX` at the moment. `Linux` and `Windows` support is forthcoming.
```
Comment (hewigovens) [inline js/README.md]:
```suggestion
`trust-wallet-core` only supports `macOS/OSX` at the moment. `Linux` and `Windows` support are forthcoming.
```

File: js/README.md [hunk 2]
```diff
@@ -26,4 +26,24 @@ from { HDWallet } from 'trust-wallet-core';
...
+
+### Bindings
+- This project uses [`node-addon-api`](https://github.com/nodejs/node-addon-api) to interface with `wallet-core` [C headers]((https://github.com/TrustWallet/wallet-core/tree/master/include/TrustWalletCore)). 
+- The `node-addon-api` functions are wrapped in TypeScript that is then used to generate a JavaScript module with type definitions.
+
+### Memory management
+
+- Please note that memory allocated with C/C++ is released once there are no references to the JavaScript object and the Node.js garbage collector tries to release it from memory.
```
Comment (hewigovens) [inline js/README.md]:
```suggestion
- Please note that memory allocated with C/C++ is released once there are no references to the JavaScript object, then Node.js garbage collector will try to release it from memory.
```

File: js/README.md [hunk 3]
```diff
@@ -26,4 +26,24 @@ from { HDWallet } from 'trust-wallet-core';
...
+
+## Other Resources
+
+- For general [`TrustWallet/wallet-core`](https://github.com/TrustWallet/wallet-core) documentation refer to [GitBooks](https://developer.trustwallet.com/wallet-core).
```
Comment (hewigovens) [inline js/README.md]:
```suggestion
- For general [`TrustWallet/wallet-core`](https://github.com/TrustWallet/wallet-core) documentation, please refer to [GitBooks](https://developer.trustwallet.com/wallet-core).
```

File: js/README.md [hunk 4]
```diff
@@ -26,4 +26,24 @@ from { HDWallet } from 'trust-wallet-core';
...
+
+- For documentation on specific modules that are avaiable. Please refer to [`TrustWallet/wallet-core`](https://github.com/TrustWallet/wallet-core) [exposed C headers](https://github.com/TrustWallet/wallet-core/tree/master/include/TrustWalletCore).
```
Comment (hewigovens) [inline js/README.md]:
```suggestion
- For documentation on specific modules that are available, please refer to [`TrustWallet/wallet-core`](https://github.com/TrustWallet/wallet-core) [exposed C headers](https://github.com/TrustWallet/wallet-core/tree/master/include/TrustWalletCore).
```

File: js/README.md (hunk 1 above)
Comment (BSathvik) [reply js/README.md]: The use of `is` is correct here since the `support` (singular) is what is forthcoming.

Comment (hewigovens) [reply js/README.md]: 👍

File: swift/Podfile.lock [hunk 5]
```diff
@@ -17,4 +17,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: cbcc5bab5b64fa9f7863e62cc106d25d3fc1dfc5
 
-COCOAPODS: 1.7.3
+COCOAPODS: 1.7.1
```
Comment (hewigovens) [inline swift/Podfile.lock]: Can you upgrade your cocoapods? Or don't commit this file

Comment (BSathvik) [reply swift/Podfile.lock]: oops, my bad. didn't mean to commit it.

File: js/README.md [hunk 6]
```diff
@@ -26,4 +24,25 @@ from { HDWallet } from 'trust-wallet-core';
...
+
+### Memory management
+
+-   Please note that memory allocated with C/C++ is released once there are no references to the JavaScript object, then Node.js garbage collector will try to release it from memory.
+
+## Other Resources
+
+-   For general [`TrustWallet/wallet-core`](https://github.com/TrustWallet/wallet-core) documentation, please refer to [GitBooks](https://developer.trustwallet.com/wallet-core).
+
+-   For documentation on specific modules that are avaiable, please refer to [`TrustWallet/wallet-core`](https://github.com/TrustWallet/wallet-core) [exposed C headers](https://github.com/TrustWallet/wallet-core/tree/master/include/TrustWalletCore).
```
Comment (hewigovens) [inline js/README.md]: typo? avaiable -> available

**Pull request merged**